lambda/README.md

```diff
@@ -1,15 +1,15 @@
 # Replacement Glue Crawler Lambda
 
-One of our primary uses of glutil is as a Glue Crawler analog for table and data that the Glue Crawler just isn't great at crawling.
+One of our primary uses of `glutil` is as a Glue Crawler analog for tables and data that the Glue Crawler cannot crawl.
 Specifically, we've run into issues running crawlers over data in non Hive-partitioned path schemas.
 To overcome this, we created a lambda that operates similarly for crawling S3 data.
 
 As with the rest of `glutil` this lambda makes the same assumptions noted in the [main readme](../README.md#built-in-assumptions).
 
-## Building a Lambda-compatible zip
+## Building a Lambda-Compatible zip
 
 The code for running the lambda is included in the `glutil` library, but just uploading the library and configuring the lambda to use a function inside the module runs into problems.
```

Comment:
@dkuntz2 Out of curiosity what is the specific problem here?

Reply:
Lambda cannot find the submodule when setting the handler to `glutil.serverless_function.handle`

``` json
{
  "errorMessage": "Unable to import module 'glutil.serverless_function'"
}
```